package.json

```diff
@@ -41,45 +41,45 @@
     "node": ">=10"
   },
   "dependencies": {
-    "commander": "6.0.0",
-    "js-reporters": "1.2.3",
-    "node-watch": "0.6.4",
-    "tiny-glob": "0.2.6"
+    "commander": "^6.0.0",
```

Comment:
For the production dependencies, we prefer to pin and audit individual packages directly. These should stay as before.

The rest of the patch is a-OK :)

Remember to re-run npm install after changing this so that the lock file is synced again.